src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+  partner = person.sex === 'm' ? 'wife' : 'husband',
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${partner}`
+        : 'I am not married'}
```

Comment:
Move it to some variable and then just use this variable in JSX, because it looks too bulky